apps/web/app/[lang]/dashboard/training-record/page.tsx

```
@@ -86,23 +86,23 @@ export default function TrainingRecordPage() {
 				<div className="flex gap-2">
 					{filter.hasOwnProperty("weight") && (
 						<Chip
-							className="text-xs py-1 px-2 border-foreground text-foreground"
+							className="text-xs py-1 px-2 border-primary/20 text-primary bg-primary/5 hover:bg-primary/10"
 							text={`${t("table.weight")}: ${filter.weight}`}
 							onClick={() => {
 								updateFilter((draft) => {
-									draft.weight = undefined;
+									delete draft.weight;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)